### ISSUE TITLE
[Tracks] - Track font size accessibility setting in device information analytics.

### ISSUE THREAD

**Issue by hafizrahman** [woocommerce/woocommerce-android#7282]
> Currently, we have some issues opened about the support for dynamic type fonts in our app [[1](https://href.li/?https://github.com/woocommerce/woocommerce-ios/issues/896), [2](https://href.li/?https://github.com/woocommerce/woocommerce-ios/issues/894)] and we are aware of the problem, but we don’t know how many users are affected by this issues.

> What we should do here, is to find a track event already implemented in the app, where we can add a new property with the font size configured by the user, for both platforms (Android and iOS). Then measure how many users are impacted by this.

Internal ref - pe5sF9-xl-p2

### How?
We can look into the device information properties added to all tracks events and find a way to add font-size information to it. Changes should be done in https://github.com/Automattic/Automattic-Tracks-Android/ instead of in this repo, and this issue is more for keeping track of things (get it?)
